src/angular/projects/spark-extras-angular-highlight-board/src/lib/spark-extras-angular-highlight-board.component.ts

```diff
@@ -104,11 +104,6 @@ export class SparkHighlightBoardComponent {
           'sprk-c-HighlightBoard--has-image sprk-c-HighlightBoard--stacked'
         );
         break;
-      case 'fullWidth':
```

Comment:
Hey @bobwise, please add some information to your PR about why there are Angular changes.